packages/nimble-components/src/table/specs/table-column-specs/table-column-anchor-hld.md

```diff
@@ -0,0 +1,121 @@
+# Anchor Table Column HLD
```

Comment:
Let's link to this document from the main table spec.

Reply:
Done